cpukit/libmisc/cpuuse/cpuusagetop.c

```diff
@@ -273,10 +273,16 @@ task_usage(Thread_Control* thread, void* arg)
               CPU_usage_Less_than(&usage, &data->usage[j]))
             continue;
         case RTEMS_TOP_SORT_REAL_PRI:
-          if (thread->Priority_node.real_priority > data->tasks[j]->Priority_node.real_priority)
+          if (
+                thread->Priority_node.real_priority >
```

Comment:
This is better,but I think you indented it too far here, and a few other places. It should be 4 white spaces deeper than the if line, e.g.:
```C
if (
    thread->Priority_node.real_priority >
    data->tasks[j]->Priority_node.real_priority
)